swagger_server/__main__.py

```diff
@@ -39,42 +39,73 @@ def get_evaluation_view(evaluation_id):
 
 @app.route('/operand', methods=['POST'])
 def add_operand_view():
-    return add_operand()
+    try:
+        post_body = request.get_json()
+        return add_operand(post_body), 200
+    except ValidationError as exc:
+        return jsonify_validation_error(exc)
 
 
 @app.route('/operand/<int:id>', methods=['DELETE'])
 def delete_operand_view(id):
+    try:
+        return delete_operand(id), 200
+    except ValidationError as exc:
+        return jsonify_validation_error(exc)
     return delete_operand(id)
```

Comment:
I think you'll wanna remove these calls that are outside of the try/except too. They shouldn't really ever get hit.